change/@itwin-property-grid-react-944e621e-779d-495c-8b49-f5a79733ce64.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Skip tests that are failing due to a backend issue",
```

Comment:
```suggestion
  "comment": "",
```

Reply:
Out of curiosity, why do you see this as a problem? The `type: "none"` will make sure it's not going to appear in the changelog.

Reply:
oh i assumed it was like rush, where even if we specify none now, when we do publish itd show up in the changelogs eventually. if thats not the case, please ignore

Reply:
I confirmed two things for changes with type "none":
- When only "none" changes exist for specific package, it's version doesn't get bumped.
- They don't get placed into `CHANGELOG.md`, only in `CHANGELOG.json`, which is fine, I think.